samples/hubspot/program.py

```diff
@@ -0,0 +1,35 @@
+"""Demonstration of AutoKitteh's HubSpot integration.
+
+This script showcases two basic operations with the HubSpot API:
+creating a new contact and listing all deals.
+"""
+
+from autokitteh.hubspot import hubspot_client
+from hubspot.crm.contacts import SimplePublicObjectInputForCreate
+
+
+hubspot = hubspot_client("hubspot_conn")
+
+
+def create_contact(event):
+    email = event.data.url.query.get("email", "meow@autokitteh.com")
+    firstname = event.data.url.query.get("firstname", "Kitty")
+    lastname = event.data.url.query.get("lastname", "Meowington")
+
+    contact_properties = {
+        "email": email,
+        "firstname": firstname,
+        "lastname": lastname,
+    }
```

Comment:
Combine these 2 blocks together - there's no need for the helper vars.

Also note that POST requests usually carry data inside them, not in the URL. See my comments in the readme file.

```suggestion
    contact_properties = {
        "email": event.data.url.query.get("email", "meow@autokitteh.com"),
        "firstname": event.data.url.query.get("firstname", "Kitty"),
        "lastname": event.data.url.query.get("lastname", "Meowington"),
    }
```